docs/source/release_notes.rst

```diff
@@ -20,6 +20,7 @@ Release Notes
         * Remove ``ColumnNameMismatchWarning`` (:pr:`777`)
         * ``get_column_dict`` does not use standard tags by default (:pr:`782`)
         * Make ``logical_type`` and ``name`` params to ``_get_column_dict`` optional (:pr:`786`)
+        * Change schema and schema_column files and Schema object to match new table-column schema structure (:pr:`789`)
```

Comment:
How about a slight simplification - `Rename Schema object and files to match new table-column schema structure`?

Reply:
This is much cleaner--changed!